config/database.php

```diff
@@ -56,7 +56,7 @@
             'collation' => 'utf8mb4_unicode_ci',
             'prefix' => '',
             'prefix_indexes' => true,
-            'strict' => true,
+            'strict' => false,
```

Comment:
Making reference of this [SO post](https://stackoverflow.com/questions/47974348/sqlstatehy000-general-error-1364-field-name-doesnt-have-a-default-value-l)
Unit tests were throwing errors as follows:
`PDOException: SQLSTATE[HY000]: General error: 1364 Field 'primary_uuid' doesn't have a default value`
This is the difference between strict usage and not.